PokerGame/PokerEngine/Card.cs

```diff
@@ -8,6 +8,7 @@ public class Card
 
     public enum Rank
     {
+        Invalid = 0, 
```

Comment:
Looking at this without any context makes it look strange why there could ever be an invalid rank.  Not saying change this immediately, but something to think about.